examples/kustomize-with-helm/servicedeployment-extras.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: deployments.plural.sh/v1alpha1
+kind: ServiceDeployment
+metadata:
+  name: plrl-04-extras
+  namespace: examples
+spec:
+  repositoryRef:
```

Comment:
this case should inflate the helm chart with kustomize and add a secret to the same kustomization (I think the kustomize-inflate-helm case really is that, and maybe can just eliminate this as a result)

Reply:
Ye, this was meant to be a separate example equivalent to the inflate one. Just another way to achieve the same result.